core-bundle/src/Resources/contao/elements/ContentAccordion.php

```diff
@@ -50,7 +50,7 @@ protected function compile()
 				->applyLegacyTemplateData($this->Template, $this->imagemargin, $this->floating);
 		}
 
-		$classes = StringUtil::deserialize($this->mooClasses);
+		$classes = StringUtil::deserialize($this->mooClasses, true) + array('', '');
```

Comment:
Isn't `array('', '')` a behavior change compared to `array(null, null)`?

Reply:
Fixed in 93139b85368a3aa735f527f91f9dba3be7353084